posts/2020/07/writing-docs-is-not-just-writing-docs.md

```diff
@@ -0,0 +1,39 @@
+<!--
+.. title: Writing docs is not just writing docs
+.. slug: writing-docs-is-not-just-writing-docs
+.. date: 2020-07-05 12:00:00 UTC-05:00
+.. author: Juanita Gomez
+.. tags: Spyder
```

Comment:
Could you change to `Spyder, documentation, Labs`?

Reply:
```suggestion
.. tags: Spyder, documentation, Labs
```